app/src/main/java/org/mozilla/focus/utils/Settings.java

```diff
@@ -115,4 +116,17 @@ public void setDefaultSearchEngine(SearchEngine searchEngine) {
     /* package */ String getPreferenceKey(int resourceId) {
         return resources.getString(resourceId);
     }
+
+    /**
+     * clear cache and return clear msg
+     * @param context
+     * @return clearCache Msg
+     */
+    public String clearCache(Context context) {
```

Comment:
Let's simply return the long diff and let view decide how it wants to format/show it.